thriftbp/server.go

```diff
@@ -91,38 +96,232 @@ func NewServer(cfg ServerConfig) (*thrift.TSimpleServer, error) {
 	return server, nil
 }
 
+// NewServerFromOptions creates a new ServerConfig instance and delegates
+// to NewServer so that it can instantiate a new thrift.TSimpleServer.
+// This is a low level api and should only be used at the recommendation
```

Comment:
style nit: add a blank line (`//`) between them.

the doc comment will be rendered similar to markdown, so line break is the same as a space, and a blank line is the same as actual line break (new paragraph).